python/katana/cpp/libgalois/Galois.pxd

```diff
@@ -3,8 +3,17 @@ from libcpp.string cimport string
 from ..libstd cimport CPPAuto
 
 
+cdef extern from "katana/ThreadPool.h" namespace "katana" nogil:
+    cppclass ThreadPool:
+        void burnPower(unsigned num)
```

Comment:
do you mind changing the name of `burnPower` to `BurnPower` (as well as `beKind`)?

Reply:
If we are renaming them, I'm gonna do more than just styling. Those names are very non-descriptive. I'll do a separate PR. This is a rush PR.

Reply:
That's fine.

(for later) I don't like that this is exposed at all, but maybe if it were wrapped in a context manager it would at least allow users to express there are a bunch of loops here that I want to run together as quickly as possible.

Reply:
I agree. I actually think this also applies to `set_active_threads` and loop annotations like `steal`. I think it's worth exploring ways to hide/automate these while also providing a fallback to control them if experts need to.